scimath/Mathematics/ClassicalStatistics.tcc

```diff
@@ -1137,8 +1137,8 @@ vector<std::map<uInt64, AccumType> > ClassicalStatistics<AccumType, DataIterator
     		vector<uInt64>::const_iterator bCounts = iCountSet->begin();
     		vector<uInt64>::const_iterator iCounts = bCounts;
     		vector<uInt64>::const_iterator eCounts = iCountSet->end();
-    		uInt dataCount = 0;
-    		uInt prevDataCount = 0;
+    		uInt64 dataCount = 0;
+    		uInt64 prevDataCount = 0;
     		uInt loopCount = 0;
```

Comment:
might as well also bump this one, there is no reason to ever use uInt for counters.